src/aap_eda/settings/default.py

```diff
@@ -204,6 +205,10 @@ def _get_boolean(name: str, default=False) -> bool:
 # See https://cfpb.github.io/django-flags/
 FLAGS = {}
 
+for feature in list(FLAGS.keys()):
```

Comment:
I think it would be better to encapsulate the logic in a function, so the logic to fetch the flag can be tested easily.

Reply:
Good suggestion, I updated